xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -31,23 +31,23 @@ If this change is undesirable, you can configure your application to use the old
 - By calling the <xref:System.AppContext.SetSwitch%2A?displayProperty=nameWithType> method as follows:
 
    ```csharp
-   AppContext.SetSwitch("System.Net.Http.useSocketsHttpHandler", false);
+   AppContext.SetSwitch("System.Net.Http.UseSocketsHttpHandler", false);
    ```
 
    ```vb
-   AppContext.SetSwitch("System.Net.Http.useSocketsHttpHandler", False)
-   ``` 
-- By defining the `System.Net.Http.useSocketsHttpHandler` switch in the *.netcore.runtimeconfig.json* configuration file:
+   AppContext.SetSwitch("System.Net.Http.UseSocketsHttpHandler", False)
+   ```
+- By defining the `System.Net.Http.UseSocketsHttpHandler` switch in the *.netcore.runtimeconfig.json* configuration file:
 
   ```json
   "runtimeOptions": {
     "configProperties": {
-        "System.Net.Http.useSocketsHttpHandler": false
+        "System.Net.Http.UseSocketsHttpHandler": false
     }
   }
   ```
-  
-- By defining an environment variable named `DOTNET_SYSTEM_NET_HTTP_USESOCKETSHTTPHANDLER` and setting it to either `false` or 0.
+
+- By defining an environment variable named `DOTNET_SYSTEM_NET_HTTP_UseSocketsHttpHandler` and setting it to either `false` or 0.
```

Comment:
was this change intentional?

Reply:
Nope, I have reverted it. Thanks for the catch. :)

Reply:
@mairaw, can this be complete? Thanks! :)